pkg/dependency/parser/java/pom/parse.go

```diff
@@ -94,10 +94,6 @@ func (p *parser) Parse(r dio.ReadSeekerAt) ([]types.Library, []types.Dependency,
 		content:  content,
 	}
 
-	remoteRepositories := root.repositories(p.servers)
-	// Add central maven repository or repositories obtained using `WithRemoteRepos` function.
-	p.remoteRepositories = append(remoteRepositories, p.remoteRepositories...)
-
```

Comment:
I specifically included this code because we need to add maven central.
Or we need to add maven central in other place

Reply:
I will check it and update the code accordingly

Reply:
@DmitriyLewen My understanding is that we don't need this code, since we are filling the p.remoteRepositories from https://github.com/aquasecurity/trivy/blob/main/pkg/dependency/parser/java/pom/parse.go#L61 

I added some logs and verified that https://repo.maven.apache.org/maven2/ is showing up in 
[analyze](https://github.com/aquasecurity/trivy/blob/main/pkg/dependency/parser/java/pom/parse.go#L313) and in [fetchPOMFromRemoteRepositories](https://github.com/aquasecurity/trivy/blob/main/pkg/dependency/parser/java/pom/parse.go#L625)

Reply:
Great. Thanks!